frontend/style.scss

```diff
@@ -513,6 +513,12 @@ button + button {
   color: $light-grey-1;
 }
 
+.redploy-button{
```

Comment:
@dancrew32 let me know if this looks good. I checked with resizing my reactive page couple ways and it works